### DISCUSSION TITLE
Push Command Error

### DISCUSSION THREAD

**Discussion by CforED** [community/community#23805] [New to GitHub]
<p>How do I remove the following Push Commend, Error Messages:</p>
<p>AzureAD+AdminUser@Surface MINGW64 /g/google/panel (master)<br>
$ git commit -m “doc commit batch” .<br>
fatal: Unable to create ‘G:/google/panel/.git/index.lock’: File exists.</p>
<p>Another git process seems to be running in this repository, e.g.<br>
an editor opened by ‘git commit’. Please make sure all processes<br>
are terminated then try again. If it still fails, a git process<br>
may have crashed in this repository earlier:<br>
remove the file manually to continue.</p>
<p>AzureAD+AdminUser@Surface MINGW64 /g/google/panel (master)<br>
$ git remote add origin <a href="mailto:git@github.com">git@github.com</a>:CforED/sway.git<br>
fatal: remote origin already exists.</p>
<p>AzureAD+AdminUser@Surface MINGW64 /g/google/panel (master)<br>
$ git push -u origin master<br>
Enter passphrase for key ‘/c/Users/AdminUser/.ssh/id_rsa’:<br>
To <a href="http://github.com" rel="nofollow noopener">github.com</a>:CforED/sway.git<br>
! [rejected] master -&gt; master (fetch first)<br>
error: failed to push some refs to ‘git@github.com:CforED/sway.git’<br>
hint: Updates were rejected because the remote contains work that you do<br>
hint: not have locally. This is usually caused by another repository pushing<br>
hint: to the same ref. You may want to first integrate the remote changes<br>
hint: (e.g., ‘git pull …’) before pushing again.<br>
hint: See the ‘Note about fast-forwards’ in ‘git push --help’ for details.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>The first error appear to be caused by trying to run multiple Git operations at the same time. Do you perhaps have several terminal windows open, and you’ve entered commands in both of them?</p>
<p>If you keep having that problem, make sure you close down all terminals except one, and then try deleting the <code>index.lock</code> file in your <code>.git</code> directory.</p>
<p><code>rm -f .git/index.lock</code></p>
<p>For the final error, it looks like the repository you’re trying to push to has changes that this version of the repository doesn’t have, so you will need to run <code>git pull</code> to get those changes and commit them, dealing with any merge conflicts, and then you should be able to push again.</p>
